src/interlocking_component/route_controller.py

```diff
@@ -243,20 +243,12 @@ def __init__(
 
     def next_tick(self, tick: int):
         self.tick = tick
-        for (
-            interlocking_route,
-            train,
-            route_length,
-        ) in self.route_queues.routes_to_be_set:
+        for (interlocking_route,) in self.route_queues.routes_to_be_set:
             # This tries to set the fahrstrasse in the interlocking.
             # The Sumo route was already set and the route was reserved.
-            was_set = self.set_interlocking_route(
-                interlocking_route, train, route_length
-            )
+            was_set = self.set_interlocking_route(interlocking_route)
             if was_set:
-                self.route_queues.routes_to_be_set.remove(
-                    (interlocking_route, train, route_length)
-                )
+                self.route_queues.routes_to_be_set.remove((interlocking_route))
         for route, train in self.route_queues.routes_to_be_reserved:
```

Comment:
does this need to be route, train or just route?

Reply:
It should be `route, train`